contrib/test/integration/ansible.cfg

```diff
@@ -58,7 +58,7 @@ gather_subset = network
 host_key_checking = False
 
 # change the default callback
-#stdout_callback = skippy
+stdout_callback = debug
```

Comment:
Is this still wanted? If yes what does it do? :innocent:

Reply:
before, we had our own stdout callback (the removed callback_plugins python code). It caused  a cyclical import problem, and now ansible ships its own stdout callback for debugging, which this line adds :)

Reply:
Alright, thanks. :+1: